core/templates/dev/head/filters.js

```diff
@@ -441,3 +441,15 @@ oppia.filter('removeDuplicatesInArray', [function() {
     });
   };
 }]);
+
+oppia.filter('stripFormatting', [function() {
+  return function(html, whitelistedImgClasses) {
+    // Strip out anything between and including <>,
+    // unless it is an img whose class includes one of the whitelisted classes.
+    var regex = new RegExp(
+      '(?!<img.*class=".*(' + whitelistedImgClasses.join('|') +
```

Comment:
I'm wondering whether ``whitelistedImgClasses`` should just be fully qualified to avoid the open captures on the sides of the classes capture group. Also, you aren't closing the class quotes.